web-api/openapi/front_office.yaml

```diff
@@ -240,7 +243,10 @@ components:
           type: boolean
           default: false
         filter:
-          $ref: '#/components/schemas/Filter'
+          description:
+            $ref: '#/components/schemas/Filter/description'
+          oneOf:
+            - $ref: '#/components/schemas/FilterCompare'
```

Comment:
i just noticed this issue in my latest pr but backported the fix here. for whatever reason the documentation generation fails to resolve the ref completely if used in a request body, eg the properties of the ref still get resolved but the description and type gets lost. interestingly resolving the ref works if used in a query param like for the other endpoint.

it would be nice to just use the ref, but i couldn't figure out if this is fixable and how to fix this, so i preferred a complete documentation over duplication. the issue seems to be rooted in the `oneOf` of the referenced schema, we could just remove it in this pr as it only contains a single item, but it will become a problem immediately in the next prs when we add more items. anybody has an idea how to make this work with just the ref?